src/v/config/configuration.cc

```diff
@@ -1974,7 +1974,20 @@ configuration::configuration()
       "Interval, in seconds, of how often a message informing the operator "
       "that unsafe strings are permitted",
       {.needs_restart = needs_restart::no, .visibility = visibility::user},
-      300s) {}
+      300s)
+  , cpu_profiler_enabled(
+      *this,
+      "cpu_profiler_enabled",
+      "Enables cpu profiling for Redpanda",
+      {.needs_restart = needs_restart::no, .visibility = visibility::user},
+      false)
+  , cpu_profiler_sample_rate_ms(
+      *this,
+      "cpu_profiler_sample_rate_ms",
+      "The maximum number of samples the cpu profiler will take per ms",
```

Comment:
maximum number of samples to take, but the value is 100ms? Shouldn't it be unitless or maybe "samples"?

The name and help indicates it is a frequency, but the value and `.min` suggests it may be a period?